server/lib/nl/detection/types.py

```diff
@@ -33,6 +33,14 @@ class Place:
   country: str = None
 
 
+@dataclass
+class Entity:
+  """Entity attributes."""
+  dcid: str
+  name: str
+  type: str
```

Comment:
clarify in comment that it a non-place entity only?

Reply:
done